### PR TITLE
Prevent pipeline fails from erroneous param validation of fasta and …

### PR DESCRIPTION
**Pull request by bjlang**
…gtf parameters if iGenomes are used

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/atacseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/atacseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/utils_nfcore_atacseq_pipeline/main.nf [hunk 1]
```diff
@@ -147,33 +148,6 @@ def validateInputParameters() {
     }
 }
 
-//
-// Validate channels from input samplesheet
-//
-def validateInputSamplesheet(input) {
```
Comment (mirpedrol) [inline subworkflows/local/utils_nfcore_atacseq_pipeline/main.nf:153]: Is this function not used?

File: assets/samplesheet_with_control.csv
Comment (mirpedrol) [inline assets/samplesheet_with_control.csv:1]: Just a heads-up that this file change, and it doesn't seem to be related to the PR, in case it was changed by mistake.

**Review by mirpedrol — APPROVED** [review]

File: assets/samplesheet_with_control.csv
Comment (bjlang) [reply assets/samplesheet_with_control.csv:1]: You're right. It's unrelated, but still correct to create coherence with assets/samplesheet.csv

File: subworkflows/local/utils_nfcore_atacseq_pipeline/main.nf (hunk 1 above)
Comment (bjlang) [reply subworkflows/local/utils_nfcore_atacseq_pipeline/main.nf:153]: No. RNAseq, which we use as reference isn't using it either.
